arviz/rcparams.py

```diff
@@ -93,23 +102,37 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
-def _validate_tuple(value):
-    """Validate value is a tuple of size 2."""
-    if value is None:
-        return value
+def _validate_float_iterable(value, length):
```

Comment:
the `_validate` functions should only have one argument, otherwise, in cases like this one the pattern to follow is the `make_validator` example, where the return value is the validator function

Reply:
Okay, I'll make the changes.